tests/MongoDB.Driver.Examples/InsertDataWithEncryptedFieldsWithAutoEncryption.cs

```diff
@@ -0,0 +1,293 @@
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using MongoDB.Bson;
+using MongoDB.Driver.Encryption;
+using Xunit;
+
+namespace MongoDB.Driver.Examples
+{
+    public class InsertDataWithEncryptedFieldsWithAutoEncryption
+    {
+        private static readonly string __connectionString = "mongodb://localhost:27017";
+        private static readonly string __sampleNameValue = "John Doe";
+        private static readonly int __sampleSsnValue = 145014000;
+
+        private static BsonDocument __sampleDocFields =
+            new BsonDocument
+            {
+                { "name", __sampleNameValue },
+                { "ssn", __sampleSsnValue },
+                { "bloodType", "AB-" },
+                {
+                    "medicalRecords",
+                    new BsonArray(new []
+                    {
+                        new BsonDocument("weight", 180),
+                        new BsonDocument("bloodPressure", "120/80")
+                    })
+                },
+                {
+                    "insurance",
+                    new BsonDocument
+                    {
+                        { "policyNumber", 123142 },
+                        { "provider", "MaestCare" }
+                    }
+                }
+            };
+
+        // https://gist.github.com/DmitryLukyanov/8417d1f5c30f12ffeb2be29056aff455#file-insertdatawithencryptedfields-cs
+        [Fact]
+        public void InsertDataWithEncryptedFields()
+        {
+            var keyVaultNamespace = CollectionNamespace.FromFullName("encryption.__keyVault");
+            var medicalRecordsNamespace = CollectionNamespace.FromFullName("medicalRecords.patients");
+
+            var keyIdBase64 = ""; // paste the generated key in Standard GuidRepresentation form here
+
+            // Construct a JSON Schema
+            var schema = JsonSchemaCreator.CreateJsonSchema(keyIdBase64);
+
+            // Construct an auto-encrypting client
+            var autoEncryptingClient = CreateAutoEncryptingClient(
+                medicalRecordsNamespace,
+                keyVaultNamespace,
+                schema);
+            var collection = autoEncryptingClient
+                .GetDatabase(medicalRecordsNamespace.DatabaseNamespace.DatabaseName)
+                .GetCollection<BsonDocument>(medicalRecordsNamespace.CollectionName);
+
+            var ssnQuery = Builders<BsonDocument>.Filter.Eq("ssn", __sampleSsnValue);
+
+            // Insert a document into the collection
+            collection.UpdateOne(ssnQuery, new BsonDocument("$set", __sampleDocFields), new UpdateOptions() { IsUpsert = true });
+            Console.WriteLine("Successfully upserted the sample document!");
+
+            // Query by SSN field with auto-encrypting client
+            var result = collection.Find(ssnQuery).Single();
+
+            Console.WriteLine("Encrypted client query by the SSN (deterministically-encrypted) field:\n" + result.ToJson());
```

Comment:
Console.WriteLine("Query by SSN (deterministically-encrypted field) using auto-encrypting client returned:\n" + result.ToJson());

Let's try to get consistent wording across all our comments and everything we write out to the Console.